build.gradle

```diff
@@ -56,6 +56,7 @@ dependencies {
     compile 'me.snowdrop:servicecatalog-client:1.0.1'
     compile 'org.apache.commons:commons-compress:1.19'
     compile 'org.apache.commons:commons-exec:1.3'
+    compile 'org.apache.commons:commons-lang3:3.9'
```

Comment:
What's the rationale for this additional dependency ?

Reply:
when switching to IC-2018.2 it wont compile complaining about commons-lang3 missing.